### DISCUSSION TITLE
Nodes with fixed address

### DISCUSSION THREAD

**Discussion by jenapkst** [openthread/openthread#9553] [Q&A] [answered]
Hi,

As per a requirement of fixed addressing of a network, I set the extended address same as IeeeEui64.
Also new mesh local assigned keeping last 8 octets as IeeeEui64.
After it, addresses are getting fixed at node ends.

Now I can see some messages as "Failed to process UDP:  Duplicated"
After several failures, the node joined the network and starts communicating.

It is normal and can be ignored?
I find there a big reformation in routing table (Leader) based on the above said restarted node.

Can you provide your expertise advice.

regards
PK Jena

**Comment by jenapkst** [comment]
Hi.
Attaching below log file for reference.
[teraterm.log](https://github.com/openthread/openthread/files/13053351/teraterm.log)

Line#158  [0000044811] [REGION UNDEF] [W] Mle-----------: Failed to process UDP: Duplicated

In this network there is one Leader and one Router. Observed single time of occurrence. For more nodes multiple times occurring.



regards
PK Jena

**Comment by jwhui** [comment, marked as answer]
> As per a requirement of fixed addressing of a network, I set the extended address same as IeeeEui64.

Thread Specification requires devices to randomly generate a new Extended Address upon factory reset.

> Now I can see some messages as "Failed to process UDP: Duplicated"

These errors are logged when the MLE Frame Counter on a received frame is less than or equal to an MLE Frame Counter on a previously received frame from the same source. If the MLE Frame Counter is equal to a previously received frame, this can simply be due to packet retransmissions. However, of the MLE Frame Counter is smaller than a previously received frame, that likely indicates an issue with maintaining the MLE Frame Counter.
